<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_interval">
        <property name="suffix">
         <string> minutes</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_timeLeft">
        <property name="text">
         <string>label_timeLeft</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSlider" name="slider_interval">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="button_setTimer">
          <property name="text">
           <string>&amp;Set timer</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_pause">
          <property name="text">
           <string>&amp;Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_stopTimer">
          <property name="text">
           <string>St&amp;op timer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>button_setTimer</tabstop>
  <tabstop>button_pause</tabstop>
  <tabstop>button_stopTimer</tabstop>
  <tabstop>slider_interval</tabstop>
  <tabstop>spinBox_interval</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
